src/containers/buttons/ViewBidsButton/ViewBidsButton.tsx

```diff
@@ -0,0 +1,93 @@
+//- Containers Imports
+import { BidList } from 'containers';
+
+//- Library Imports
+import { Bid } from '@zero-tech/zns-sdk/lib/zAuction';
+import { Maybe, Metadata } from 'lib/types';
+import { Domain } from '@zero-tech/zns-sdk/lib/types';
+
+//- Components Imports
+import { FutureButton, Overlay, TextButton } from 'components';
+
+//- Constants Imports
+import { LABELS } from './ViewBidsButton.constants';
+
+interface ViewBidButtonProps {
+	bids?: Bid[];
+	domain?: Domain;
+	buttonText?: string;
+	className?: string;
+	isTextButton?: boolean;
+	refetch: () => void;
+	style?: React.CSSProperties;
+	highestBid?: number;
+	isLoading?: boolean;
+	domainMetadata?: Maybe<Metadata>;
+	setIsViewBidsOpen?: (state: boolean) => void;
+	isViewBidsOpen?: boolean;
+}
+
+const ViewBidsButton = ({
+	bids,
+	domain,
+	buttonText,
+	className,
+	isTextButton,
+	refetch,
+	style,
+	highestBid,
+	isLoading,
+	domainMetadata,
+	setIsViewBidsOpen,
+	isViewBidsOpen,
+}: ViewBidButtonProps) => {
+	const onClose = () => {
+		if (setIsViewBidsOpen) {
+			setIsViewBidsOpen(false);
+		}
+	};
+
+	const onClick = () => {
+		if (setIsViewBidsOpen) {
+			setIsViewBidsOpen(true);
+		}
+	};
+
+	const onAccept = () => {
+		refetch();
+		onClose();
+	};
+
+	return (
+		<>
+			{isViewBidsOpen && domainMetadata && bids && (
+				<Overlay open={isViewBidsOpen} centered onClose={onClose}>
+					<BidList
+						bids={bids}
+						domain={domain}
+						domainMetadata={domainMetadata}
+						onAccept={onAccept}
+						highestBid={String(highestBid)}
```

Comment:
Struggling to format the ```highestBid``` value. When using existing format utils it will crash the browser.
<img width="729" alt="Screenshot 2022-04-04 at 21 39 16" src="https://user-images.githubusercontent.com/39112648/161628144-cd1e751e-68b4-4965-8502-faaff917e222.png">

Reply:
It looks like the `<Details>` component in `AcceptBid` is looking for bid amount in wei as a string. Should be able to send it as `parseEther(highestBid.toString()).toString()`. Could be worth changing the variable to `highestBidAsWei`.